build.sh

```diff
@@ -148,6 +148,20 @@ update_ssh_config() {
   echo -e "\nHost *\n\tStrictHostKeyChecking no" >> $HOME/.ssh/config
 }
 
+check_uuid_module(){
+  if [ $? $(python -c "import uuid") != 0 ]; then
+    current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Does this interfere with the location set by update_dir? We also have $PROGDIR defined at the top of the file. Can we not use that instead of trying to figure it out again?